sku.routes.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+const partition = require('lodash/partition');
+
+const componentsPath = path.join(__dirname, 'lib/components/index.js');
+const componentsSource = fs.readFileSync(componentsPath, 'utf-8'); // eslint-disable-line no-sync
+const components = componentsSource
+  .match(/default as [A-Za-z]+/g)
+  .map(x => x.replace('default as ', ''))
+  .sort();
+const [iconNames, componentNames] = partition(components, x =>
+  /.+Icon$/i.test(x)
+);
+
+module.exports = [
+  { route: '/', name: 'home' },
+  ...componentNames.map(name => ({ route: `/components/${name}/`, name })),
```

Comment:
Did you consider having an entry for each component?

Reply:
Yeah—didn't seem worth the complexity in this case. Either way, I didn't see it as a blocker to migrate to the new API.